benchmark_test.go

```diff
@@ -218,3 +220,76 @@ func TestBenchmarkStatsPerPeer(t *testing.T) {
 	})
 	assert.Equal(t, want, statsServer.Aggregated(), "unexpected stats")
 }
+
+// Method that tests the 'Format' option of Benchmark Options
+func TestBenchmarkOutput(t *testing.T) {
+	// Testing both plaintext and JSON output
+	tests := []struct {
+		opts BenchmarkOptions
+	}{
+		{
+			opts: BenchmarkOptions{
+				Format: "json",
+			},
+		},
+		{
+			opts: BenchmarkOptions{
+				Format: "plaintext",
```

Comment:
can we add a test case with a different format, to test the behavior that we default to plain text